### PR TITLE
PHPCS: Include all files by default

### PR DESCRIPTION
**Pull request by obenland**
I feel a bit bad that I got myself suckered into making changes beyond just phpcs-stuff, but they should be straightforward.

## Proposed changes:

* Switches to covering all files with a list of exceptions.
* Fixes phpcs issues in newly included files.
* Fixes some phpcs issues that had inline-ignores.
* Updates some template-related syntax.

### CODE REVIEW

File: includes/functions.php [hunk 1]
```diff
@@ -1012,8 +1012,7 @@ function ( $enclosure ) {
 function get_comment_ancestors( $comment ) {
 	$comment = \get_comment( $comment );
 
-	// phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
-	if ( ! $comment || empty( $comment->comment_parent ) || $comment->comment_parent == $comment->comment_ID ) {
+	if ( ! $comment || empty( $comment->comment_parent ) || (int) $comment->comment_parent === (int) $comment->comment_ID ) {
```
Comment (obenland) [inline includes/functions.php:1015]: This should be fine, right?

File: includes/handler/class-announce.php [hunk 2]
```diff
@@ -33,27 +33,25 @@ public static function init() {
...
-	 * @param int                            $user_id  The id of the local blog-user.
-	 * @param \Activitypub\Activity\Activity $activity The activity object.
-	 *
-	 * @return void
+	 * @param array                          $announcement The activity-object.
+	 * @param int                            $user_id      The id of the local blog-user.
+	 * @param \Activitypub\Activity\Activity $activity     The activity object.
 	 */
-	public static function handle_announce( $array, $user_id, $activity = null ) { // phpcs:ignore Universal.NamingConventions.NoReservedKeywordParameterNames.arrayFound
+	public static function handle_announce( $announcement, $user_id, $activity = null ) {
```
Comment (obenland) [inline includes/handler/class-announce.php:40]: Is this a good name for this variable?

File: templates/settings.php [hunk 3]
```diff
@@ -184,19 +189,21 @@
...
 								<?php \esc_html_e( 'Automatically publish items of the selected post types to the fediverse:', 'activitypub' ); ?>
 
-								<?php $post_types = \get_post_types( array( 'public' => true ), 'objects' ); ?>
-								<?php $support_post_types = \get_option( 'activitypub_support_post_types', array( 'post' ) ) ? \get_option( 'activitypub_support_post_types', array( 'post' ) ) : array(); ?>
 								<ul>
-								<?php // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited ?>
-								<?php foreach ( $post_types as $post_type ) { ?>
+								<?php
+								$post_types           = \get_post_types( array( 'public' => true ), 'objects' );
+								$supported_post_types = (array) \get_option( 'activitypub_support_post_types', array( 'post' ) );
```
Comment (obenland) [inline templates/settings.php:192]: Is this what line 188 was supposed to do? Make sure it's an array?

File: includes/transformer/class-factory.php [hunk 4]
```diff
@@ -10,21 +10,21 @@
...
- * Transformer Factory
+ * Transformer Factory.
  */
 class Factory {
 	/**
 	 * Get the transformer for a given object.
 	 *
-	 * @param  mixed $object The object to transform.
+	 * @param mixed $data The object to transform.
 	 * @return Base|WP_Error The transformer to use, or an error.
```
Comment (pfefferle) [inline includes/transformer/class-factory.php]:
```suggestion
	 * @param mixed $data The object to transform.
	 *
	 * @return Base|WP_Error The transformer to use, or an error.
```

**Review by pfefferle — DISMISSED** [review]
nice, thanks a lot!

**Review by pfefferle — APPROVED** [review]

**Pull request merged**